src/router.js

```diff
@@ -97,4 +97,13 @@ export default class Router {
   forward() {
     window.history.forward();
   }
+
+  pop() {
+    if (this.currentPath() !== '/') {
```

Comment:
I am not sure if this implementation assumes too much.

what about something like:

```js
var matches = this.currentPath().match(/.+(\/.+\/?)/);
if (matches) {
  this.navigate(matches[1]);
}
```